### PR TITLE
Release version 2.3.0

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,8 @@
  - Fixed a bug where upgrading plugins with Manticore threw an error and sometimes corrupted installation ([#4818](https://github.com/elastic/logstash/issues/4818)).
  - Removed milestone warning that was displayed when the `--pluginpath` option was used to load plugins ([#4562](https://github.com/elastic/logstash/issues/4562)).
  - Upgraded to JRuby 1.7.24.
+ - Reverted default output workers to 1. Perviously we had made output workers the same as number of pipeline
+   workers ([#4877](https://github.com/elastic/logstash/issues/4877)).
```
Comment (ph) [inline CHANGELOG.md]: Do we want to include a note about **logstash-core-plugin-api** in the changelog?

File: logstash-core-plugin-api/lib/logstash-core-plugin-api/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # encoding: utf-8
-# 1.2.0 tracks 2.3.0.snapshot5
-LOGSTASH_CORE_PLUGIN_API = "1.2.0"
+# 1.3.0 tracks LS 2.3.0
```
Comment (ph) [inline logstash-core-plugin-api/lib/logstash-core-plugin-api/version.rb]: good idea +1

File: CHANGELOG.md (hunk 1 above)
Comment (suyograo) [inline CHANGELOG.md]: Nah, its not user facing, so I'd prefer to leave it out

**Comment by ph** [comment]
LGTM

**Pull request merged**